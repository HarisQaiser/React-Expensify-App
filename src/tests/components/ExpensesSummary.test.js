import React from 'react';
import { shallow } from 'enzyme';
import ExpensesSummary  from '../../components/ExpenseDashboardPage';
import Enzyme from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

Enzyme.configure({
    adapter: new Adapter()
})


test('should render ExpensesSummary with 1 expense', () =>{
  const wrapper = shallow(<ExpensesSummary 
                            expenseCount={1} expensesTotal={235}/>);
  expect(wrapper).toMatchSnapshot();                          
});


test('should render ExpensesSummary with multiple expense', () =>{
    const wrapper = shallow(<ExpensesSummary 
                      expenseCount={24} expensesTotal={2351231534}/>);
    expect(wrapper).toMatchSnapshot(); 
});
