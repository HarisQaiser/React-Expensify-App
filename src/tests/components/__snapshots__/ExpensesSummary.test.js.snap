// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpensesSummary with 1 expense 1`] = `
<div>
  <Connect(ExpensesSummary) />
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;

exports[`should render ExpensesSummary with multiple expense 1`] = `
<div>
  <Connect(ExpensesSummary) />
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
